src/content/nords/blue_axe/ideas.txt

```diff
@@ -0,0 +1,72 @@
+new idea_group rift_ideas = {
+	name = "Blue Axe"
+	category = national
+	start = {
+		garrison_size = 0.2
+		infantry_power = 0.10
+	}
+	bonus = {
+		overextension_impact_modifier = -0.1
+	}
+	trigger = {
+		tag = blue_axe
+	}
+	ideas = {
+		{
+			name = "Glacial Vanguard"
+			desc = "Descendants of the mighty Atmoran axe-warriors, the people of Blue-Axe have long stood as the Glacial Vanguard against the encroaching threats of the frozen north. Fierce and unyielding, their warriors are renowned for their expertise in combating the chilling cold and facing the relentless hordes that lurk in the icy wilderness."
+			modifier = {
+				shock_damage = 0.05
+				shock_damage_received = -0.1
+			}
+		}
+		{
+			name = "Sea-Thanes' Dominion"
+			desc = "The rise of the Sea-Thanes has transformed Blue-Axe into a maritime power to be reckoned with. These independent lords of the sea command fleets that dominate the northern waters, securing trade routes and fending off any would-be invaders. Blue-Axe's dominion over the icy waves ensures prosperity and control over vital maritime trade routes."
+			modifier = {
+				cannons_for_hunting_pirates_in_fleet = 1.2
+				global_ship_trade_power = 0.2
+				galley_power = 0.2
+			}
+		}
+		{
+			name = "Rieklings' Bane Legacy"
+			desc = "Rooted in the ancient history of Blue-Axe, the Rieklings' Bane Legacy commemorates the heroic deeds of the Atmoran forebears who purged the northern reaches of Winterhold from the wretched Rieklings. The memory of this cleansing campaign instills a deep-seated determination to resist the resurgence of these malevolent creatures."
+			modifier = {
+				native_uprising_chance = -0.25
+				native_assimilation = 0.25
```

Comment:
Neither modifier is really that useful since there isn't anything to colonize. Maybe simple prestige?